client/components/plans/plan-actions/index.jsx

```diff
@@ -251,37 +264,44 @@ module.exports = React.createClass( {
 		analytics.ga.recordEvent( 'Upgrades', 'Clicked Current Plan' );
 	},
 
-	getTrialPlanHint: function() {
-		var remainingDays = this.moment(
+	getTrialPlanHint() {
```

Comment:
I have no idea how to test this part. In my opinion this code can't be reached with current plans setup (we always show plan overview page for free trials).